src/lib/uTop_private.ml

```diff
@@ -9,6 +9,28 @@
 
 open Lwt_react
 
+module DefaultPaths = struct
+  let ( / ) = Filename.concat
+  let xdg = Xdg.create ~env:Sys.getenv_opt ()
+
+  (* Fallback to legacy filename *)
+  let resolve legacy fn =
```

Comment:
Can you use named arguments here?

Reply:
Is `DefaultPaths` idiomatic OCaml, or should the module name be `Default_Paths`? I realize I may make some unidiomatic decisions, since I'm more used to SML.